docs/en/observability/slo-create.asciidoc

```diff
@@ -190,8 +190,8 @@ For more information, see <<slo-burn-rate-alert, Create an SLO burn rate rule>>.
 [[slo-dashboard]]
 = Add an SLO Overview panel to a custom dashboard
 
-After you've created your SLO, you can monitor it from the _SLOs_ page in Observability,
-but you can also add an _SLO Overview_ panel to a custom dashboard.
+After you've created your SLO, you can monitor it from the _SLOs_ page in Observability. If you have
+the {subscriptions}[appropriate license], you can also add an _SLO Overview_ panel to a custom dashboard.
```

Comment:
Thanks! I like this wording as it ensures the subscriptions page remains the source of truth for the type of license that is needed.